### ISSUE TITLE
Visualizer blank

### ISSUE THREAD

**Issue by Josh5A** [Mossy1022/Smart-Connections-Visualizer#12]
The visualizer isn't showing anything for me, even after adjusting min relevance to 0. 

So I tested it out in a sandbox with no other plugins running just to rule out plugin conflicts:

![image](https://github.com/user-attachments/assets/d279dc7b-011b-46dd-b5fd-93ea8352c497)

**Comment by Mossy1022** [comment]
Hey @Josh5A!  So sorry about that.  There was an update in the smart connections app that this depends on.  Please update to v1.0.17 and see if that resolves your issue 😁

**Comment by daguito81** [comment]
I have Smart Connections 2.1.100 and Smart Connections Visualizer 1.0.17  and the visualizer is completely blank  and shows me the error that no nodes or links

**Comment by Mossy1022** [comment]
Hey @daguito81, apologies for the delay! I have confirmations that it works with Smart Connections v2.2.83 and Visualizer v1.0.18.  Hoping you're not a special case, but let me know if you have issues after that.

**Comment by PhotoVisionProject** [comment]
Unfortunately, I’m dealing with the same issue: **a blank page!**

I use: 

obsidian v.1.8  (Installer version: v1.7.5)
Smart Connections:  2.3.47
Smart Connections Visualizer: 1.0.19

**Comment by Ozymale** [comment]
Just installed. Unfortunately, I’m also dealing with the same issue: a blank page. Would be grateful for your attention to this extension, as I really connect with the concept.

I use:

obsidian v.1.7.7 (Installer version: v1.5.12)
Smart Connections: 2.3.47
Smart Connections Visualizer: 1.0.19

**Comment by IMB11** [comment]
Hi, im encountering this issue as well, same versions as @Ozymale

**Comment by IMB11** [comment]
Shame, as I found this plugin incredibly useful.

**Comment by mimkorn** [comment]
Same problem here.

**Comment by aldrinjenson** [comment]
same issue. any fix?

**Comment by paulhoff** [comment]
Not working for me either.
obsidian v.1.8.0 (Installer version: v1.7.7)
Smart Connections : 2.3.47
Smart Connections Visualizer: 1.0.19

**Comment by Mossy1022** [comment]
Hey Guys! So sorry I'd been busy for the holidays.  I'm assuming there was an update to Smart Connections that caused it to break given these recent comments. I will check it out!

**Comment by Mossy1022** [comment]
Yes I see the issue, the function for finding connections and properties of notes have changed/moved somewhere.  Checking in with Brian, but should have a resolution soon!

**Comment by Mossy1022** [comment]
@PhotoVisionProject  @paulhoff  @aldrinjenson  @mimkorn @IMB11 @Ozymale 

Hey Everyone! I just made a new release v1.0.20 to account for the new changes with Smart Connections.  I believe it should work with that.  Let me know if this resolves it for you 😁

**Comment by PhotoVisionProject** [comment]
I was unable to update to a new version or install the plugin from scratch.

Obsidian Error Message: Failed to install the plugin "Smart Connections Visualizer."

**Comment by IMB11** [comment]
> Obsidian Error Message: Failed to install the plugin "Smart Connections Visualizer."

**Comment by aldrinjenson** [comment]
Worked now! Thanks @Mossy1022

**Comment by PhotoVisionProject** [comment]
SUPER! IT WORKS! THANKS A LOT! @Mossy1022
